.github/workflows/conformance-aks-v1.10.yaml

```diff
@@ -136,6 +136,16 @@ jobs:
           fi
 
           CILIUM_INSTALL_DEFAULTS="--cluster-name=${{ env.name }} \
+            --chart-directory=install/kubernetes/cilium \
+            --helm-opts=image.repository=quay.io/cilium/cilium-ci \
+            --helm-opts=image.tag=${SHA} \
+            --helm-opts=operator.image.repository=quay.io/cilium/operator \
+            --helm-opts=operator.image.suffix=-ci \
```

Comment:
Is there a specific reason we're using the suffix only for Operator image and not other images? For consistency I think it's better if we pick one or the other for all images.

Reply:
This is because the operator images are automagically suffixed by cloud provider version, which caused `-ci` suffix to clash (we ended up with `operator-ci-aws` when we wanted `operator-aws-ci`).